README.md

```diff
@@ -1,7 +1,9 @@
 # ivvv
+[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/kmader/ivvv/patch-2)
```

Comment:
In order to merge this into imaging-tools:master shouldn't all the gh/kmader lines be changed to gh/imaging-tools ?